sigmastate/src/main/scala/sigmastate/serialization/DataJsonEncoder.scala

```diff
@@ -95,7 +94,9 @@ object DataJsonEncoder {
     case tOpt: SOption[a] =>
       val opt = v.asInstanceOf[tOpt.WrappedType]
       if (opt.isDefined) {
-        encodeData(opt.get, tOpt.elemType)
+        // save the single value as an array with one item
+        val valueJson = encodeData(opt.get, tOpt.elemType)
+        Json.fromValues(Array(valueJson))
```

Comment:
Seems like JSON format has been changed?  Why? Why not covered in PR description?

Reply:
see above